src/twisted/conch/ssh/transport.py

```diff
@@ -279,6 +281,24 @@ def verify(self, seqid, data, mac):
         outer = hmac.HMAC(self.inMAC.key, data, self.inMAC[0]).digest()
         return hmac.compare_digest(mac, outer)
 
+    def isOutMACETM(self) -> bool:
+        """
+        @return True if outgoing MAC is ETM.
+        """
+        return self.outMACType.endswith(b"-etm@openssh.com")
```

Comment:
Is `outMACType` intended to be a mutable attribute? Perhaps this could be computed in `__init__`?

Reply:
True. Not at `__init__` , but at `setKeys()`. I will do that. Thanks